Userland/Applications/Piano/WaveWidget.cpp

```diff
@@ -18,7 +18,7 @@ WaveWidget::WaveWidget(TrackManager& track_manager)
 
 int WaveWidget::sample_to_y(int sample) const
 {
-    constexpr int nice_scale_factor = 4;
+    constexpr int nice_scale_factor = 1;
```

Comment:
*Scale* factor... 1?

Reply:
We might need 0.8 in the future. Let's just leave it for now, k?